tests/test_csv.py

```diff
@@ -78,6 +80,22 @@ def test_table_csv_with_nullable_labels(app_client):
     assert EXPECTED_TABLE_WITH_NULLABLE_LABELS_CSV == response.text
 
 
+@pytest.mark.xfail
```

Comment:
Don't forget to remove the `xfail` (and the `import pytest` earlier) when I get this test to pass.